src/test/resources/pages/instructorFeedbackDeleteSuccessful.html

```diff
@@ -1014,7 +1014,7 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
                     </a>
                   </li>
                   <li>
-                    <a class="session-publish-for-test" data-fsname="private session of characters1234567 #" data-sending-published-email="true" href="/page/instructorFeedbackPublish?courseid=CFeedbackUiT.CS2104&fsname=private+session+of+characters1234567+%23&next=%2Fpage%2FinstructorFeedbackSessionsPage&user=CFeedbackUiT.instructor&token=${sessionToken}">
+                    <a class="session-publish-for-test" data-fsname="private session of characters1234567 #" data-sending-published-email="true" disabled="" href="/page/instructorFeedbackPublish?courseid=CFeedbackUiT.CS2104&fsname=private+session+of+characters1234567+%23&next=%2Fpage%2FinstructorFeedbackSessionsPage&user=CFeedbackUiT.instructor&token=${sessionToken}">
```

Comment:
Revert this; it's causing the build failure. This test has a dependency on the browser time zone; it'll fail when it (significantly?) differs from UTC.